src/Runner.js

```diff
@@ -88,6 +88,16 @@ export default class Runner extends EventEmitter {
     if (this.options.args) {
       args.push(...this.options.args.args);
     }
+    if (this.workspace.useDashedArgs) {
+      args.forEach((arg, index) => {
```

Comment:
This code is not wrong, but usually, we use `map` for transformation operations. This might be more of a style and convention rather than correctness...